ClientApp/src/Pages/ConfirmSighting/ConfirmSighting.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { FormEvent, useState, useEffect } from "react";
+import { useParams } from "react-router";
+import { getSighting } from "../../Api/apiClient";
+import "./ConfirmSighting.scss";
+
+export function ConfirmSightingForm(): JSX.Element {
+  const [status, setStatus] = useState("LOADING");
```

Comment:
Could probably change this to a boolean of isLoading or something instead of matching on a string here :)